tests/Populator/PropertyMappingPopulatorTest.php

```diff
@@ -35,4 +36,63 @@ public function test_populate_default_value(): void
 
         self::assertSame('default', $person->getFullName());
     }
+
+    public function test_populate_null_safety(): void
+    {
+        $populator = new PropertyMappingPopulator(
+            'fullName',
+            'fullName',
+            null,
+            null,
+            null,
+            true
+        );
+        $user = (new User())->setFullName(null);
+        $person = new Person();
+        $person->setFullName('old Name');
+
+        $populator->populate($person, $user);
+
+        self::assertSame('old Name', $person->getFullName());
+    }
+
+    public function test_populate_with_dot_operator(): void
+    {
+        $populator = new PropertyMappingPopulator(
+            'placeOfResidence',
+            'address.city',
+            null,
+            null,
+            null,
+            true
+        );
+        $user = (new User())->setAddress((new Address())->setCity('Bremen'));
+
+        $person = new Person();
+
+        $populator->populate($person, $user);
+
+        self::assertSame('Bremen', $person->getPlaceOfResidence());
+    }
+
+    // This functionality will be automatically possible with Symfony 6.2 or higher.
+    // public function test_populate_with_dot_operator_and_null_safety(): void
+    // {
+    //    $populator = new PropertyMappingPopulator(
+    //        'placeOfResidence',
+    //        'address?.city',
+    //        null,
+    //        null,
+    //        null,
+    //        true
+    //    );
+    //    $user = (new User())->setAddress(null);
+    //
+    //    $person = new Person();
+    //    $person->setPlaceOfResidence('Old City');
+    //
+    //    $populator->populate($person, $user);
+    //
+    //    self::assertSame('Old City', $person->getPlaceOfResidence());
+    // }
```

Comment:
Does it work when you annotate the test with `/** @requires function \Symfony\Component\PropertyAccess\PropertyPath::isNullSafe */` instead of commenting it out?

Reply:
I'll try that.

Reply:
Works. Yeah!